lib/components/parallax_component.dart

```diff
@@ -117,10 +116,13 @@ class ParallaxLayer {
         break;
     }
 
-    final dx = _scroll.x * _imageSize.x;
-    final dy = _scroll.y * _imageSize.y;
-
-    _paintArea = Rect.fromLTWH(-dx, -dy, _paintArea.width, _paintArea.height);
+    final Vector2 scrollPosition = Vector2.copy(_scroll)..multiply(_imageSize);
```

Comment:
whats the difference between Vector2.copy and vector2.clone()?

Reply:
Nothing really, I guess we should settle for `clone` since that is used in the most places.